### PR TITLE
[ENG-2719] Added new translation strings for registries that are in the initial ReviewsState

### PR DESCRIPTION
**Pull request by bp-cos**
-   Ticket: [ENG-2719]
-   Feature flag: n/a

## Purpose

Added new translation strings for registries that are in the initial ReviewsState

## Summary of Changes

Three translations string were missing. These string were provided by the server.

## Screenshot(s)
Screenshot 2023-01-24 at 11 06 43 AM
Screenshot 2023-01-24 at 10 50 15 AM

## Side Effects

None

## QA Notes

The registry needs to be in an "initial" state.

[ENG-2719]: https://openscience.atlassian.net/browse/ENG-2719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

File: translations/en-us.yml [hunk 1]
```diff
@@ -1363,6 +1363,12 @@ registries:
     form_view:
         none_selected: 'None selected'
     overview:
+        initial:
+            text: 'Pending'
+            short_description: 'Pending admin contributor approval'
+            long_description: 'This registration is awaiting approval by all admin contributors or after 48 hours has passes. An email will notify all registration contributors of the decision.
```
Comment (futa-ikeda) [inline translations/en-us.yml:1369]: "after 48 hours has passed"

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
One small typo

File: translations/en-us.yml [hunk 2]
```diff
@@ -1363,6 +1363,12 @@ registries:
     form_view:
         none_selected: 'None selected'
     overview:
+        initial:
+            text: 'Pending'
+            short_description: 'Pending admin contributor approval'
+            long_description: 'This registration is awaiting approval by all admin contributors or after 48 hours has passed. An email will notify all registration contributors of the decision.
+            <br><br>
+            If the registration is pending after 48 hours has passed, contact support at support@osf.io.'
```
Comment (brianjgeiger) [inline translations/en-us.yml:1371]: This should probably be the configured support email. Is that feasible to do without having to mess with a lot of structure for the generalized messages?

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
One thing, if it's not a rabbit hole.

**Review by brianjgeiger — DISMISSED** [review]

**Review by futa-ikeda — APPROVED** [review]

File: translations/en-us.yml [hunk 3]
```diff
@@ -1393,7 +1393,14 @@ registries:
     form_view:
         none_selected: 'None selected'
     overview:
+        initial:
+            text: 'Pending'
+            short_description: 'Pending admin contributor approval'
```
Comment (chth0n1x) [inline translations/en-us.yml:1398]: Brian, I believe Admin should be either capitalized or spelled out fully as 'administrator'. This shorthand is slightly informal.

**Review by chth0n1x — APPROVED** [review]
These changes look good. I would like to see the spelling of Admin updated, but that is up to team input. Approving.

**Pull request merged**